package.json

```diff
@@ -9,27 +9,29 @@
   },
   "dependencies": {
     "@react-native-community/masked-view": "0.1.10",
-    "@react-navigation/bottom-tabs": "^5.9.1",
-    "@react-navigation/native": "^5.7.5",
-    "@react-navigation/stack": "^5.9.2",
-    "expo": "^39.0.0",
-    "expo-linking": "^1.0.4",
+    "@react-navigation/bottom-tabs": "^6.3.1",
+    "@react-navigation/native": "^6.0.10",
+    "@react-navigation/stack": "^6.2.1",
+    "add": "^2.0.6",
+    "expo": "41.0.0",
```

Comment:
Is there a reason not to use the newest expo version? Also,could you remove the unnecessary libs like `add` or `yarn`?